src/main/kotlin/domain/game/LottoGame.kt

```diff
@@ -0,0 +1,47 @@
+package domain.game
+
+import domain.lotto.PurchasedLotto
+import domain.lotto.WinningLotto
+import domain.lotto.number.LottoNumber
+import domain.rank.Rank
+import kotlin.math.floor
+
+class LottoGame(
+    private val lottoMachine: LottoMachine
+) {
+    private lateinit var winningLotto: WinningLotto
+    private var bonusNumber: LottoNumber? = null
+
+    fun initWinningLottoNumbers(_lottoNumbers: List<Int>, _bonusNumber: Int) {
```

Comment:
```suggestion
    fun initWinningLottoNumbers(lottoNumbers: List<Int>, bonusNumber: Int) {
```
대체로 변수명 앞에 언더바를 붙이는 경우는, backing property를 구현할 때 사용합니다.
변수명 앞에 언더바를 제거해보면 어떨까요?

ref. https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties